### PR TITLE
hide button copy when flash is none

### PR DESCRIPTION
**Pull request by dtagdev1**

### CODE REVIEW

**Comment by dtagdev1** [comment]
hi @asafdav, I have change commit and README. Now, you can merge .

File: README.md [hunk 1]
```diff
@@ -55,18 +55,21 @@ ng-clip is available at [cdnjs](http://www.cdnjs.com/libraries/ng-clip)
   ```html
   <a href="" clip-click"getHtmlToCopy()" clip-copy-mime-type="text/html">Copy HTML</a>
   ```
-  
+
 7. You can also optionally provide a fallback function that gets called if flash is unavailable:
   ```html
   <a href="" clip-click-fallback="fallback(copy)" clip-copy="getTextToCopy()" clip-click="doSomething()">Copy</a>
   ```
+8. set `auto-hide-on-no-flash = true`, it is used to hide button copy when browser flash is none, default `auto-hide-on-no-flash = false`, example:
```
Comment (asafdav) [inline README.md]: You cut section 7 in the middle (If the fallback function...), can you please move your section to the end of the list  ?

**Comment by asafdav** [comment]
Hi, please see my comment, you don't have to create a new pull request, you can just squash your commits and push it again. 
Thanks !

**Comment by dtagdev1** [comment]
sorry @asafdav . Thanks you !

**Comment by asafdav** [comment]
@dtagdev1 awesome, thank you!

**Pull request merged**